src/main/java/appeng/api/util/OnCompleteListener.java

```diff
@@ -1,10 +1,10 @@
-package appeng.me.cluster.implementations;
+package appeng.api.util;
 
 import java.io.Serializable;
 
 @FunctionalInterface
 public interface OnCompleteListener<A1, A2, A3> extends Serializable {
```

Comment:
This type should also be names `CraftCompleteListener` (to math the other two).

Reply:
Done